src/players/Vidyard.js

```diff
@@ -50,6 +53,7 @@ export default class Vidyard extends Component {
 
   stop () {
     window.VidyardV4.api.destroyPlayer(this.player)
+    delete this.player
```

Comment:
Is this really necessary?

Reply:
@CookPete I think it's logical. If you destroy object and `this.player` doesn't exist, you don't need to keep reference to non-existing object.

Reply:
I’ve merged this but left this line out for the time being. I’m not a fan of the delete statement and would rather be consistent with the rest of the codebase (which avoids calling delete anywhere)

Reply:
ok clear :) any case, I checked new release with fix and works well